### ISSUE TITLE
Latest version (2.23.2) can't download newrelic agent zip if version is set

### ISSUE THREAD

**Issue by petergphillips** [djoos-cookbooks/newrelic#276]
Thanks very much for the quick feedback and version fix in https://github.com/djoos-cookbooks/newrelic/issues/275.  However now getting 
```     ArgumentError
    -------------
    wrong number of arguments (0 for 1)
    
    Cookbook Trace:
    ---------------
    /var/chef/cache/cookbooks/newrelic/providers/agent_java.rb:45:in `agent_jar'
```
looking at the code I think it should be
```               "newrelic-java-#{new_resource.version}.zip"
``` instead as `version` isn't declared at that point.

**Comment by djoos** [comment]
Yeah, haste being the enemy of perfection, I also spotted now that the filename-variable is not actually used in the url.

I'll get on the case...

**Comment by djoos** [comment]
...fixed in 2.23.3, thanks in advance for your feedback!
